packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -31,6 +31,42 @@
   height: 100%;
 }
 
+@mixin content-alignment {
+  &--start {
+    align-items: start;
+  }
+
+  &--center {
+    align-items: center;
+  }
+
+  &--end {
+    align-items: end;
+  }
+}
+
+@mixin text-alignment {
+  &--start {
+    align-items: start;
+    text-align: start;
+  }
+
+  &--center {
+    align-items: center;
+    text-align: center;
+  }
+
+  &--end {
+    align-items: start;
+    text-align: start;
+  }
+}
+
+@mixin reset-alignment {
+  align-items: start;
```

Comment:
Using `start` instead of `initial` fixes the stretching of the logo in portrait.